### DISCUSSION TITLE
Antifurto integrabile in HA

### DISCUSSION THREAD

**Discussion by Salsalove** [indomus/forum#1320] [Hardware & Firmware]
Scrivo qui, magari qualcuno potrebbe consigliarmi.
Sono in procinto di acquistare un allarme per la casa nuova e sto cercando qualcosa di già settato e funzionante con app dedicata ma che si possa tranquillamente integrare in HA.
Da questa richiesta escludo già a proporvi Antifurto365 le cui nuove centraline non sono compatibili con l'integrazione ufficiale e nemmeno con quella custom...ed escluderei anche Verisure che pare essere abbastanza caro.
Inutile dire che quando parlo di "integrare" mi riferisco sia alla centralina (e quindi di poter gestire i vari comandi da HA) che tutti i sensori movimento e porta/finestra che possono così essere utilizzato anche per altri scopi.

Qualche consiglio legato alla propria esperienza? Grazie in anticipo!

**Comment by KC-inDomus** [comment]
ciaom intanto ti suggeriamo [di leggere questo](https://indomus.it/focus/panoramica-i-sistemi-dallarme-la-domotica-e-home-assistant/).
